src/util/sample.cpp

```diff
@@ -183,7 +183,6 @@ void SampleUtil::applyAlternatingGain(CSAMPLE* pBuffer, CSAMPLE gain1,
         return;
     }
 
-    // note: LOOP VECTORIZED.
```

Comment:
Interesting. Maybe we need to add different comments for different compilers? 
The note was mainly because the loop shall not be touched to not disallow vectorization. 
I will investigate the issue.